server_tests/test/clientTests.js

```diff
@@ -787,89 +789,151 @@ describe('RUST API TEST SUITE', function () {
     if (!isOSC2) {
       return this.skip()
     }
+    var expectedImageCount
+    var expectedVideoCount
+    var totalEntryCount
 
     before(function () {
       // delete exisiting files on SD card, if any
+      return Utility.deleteAllImages() //should have a new utility deleteAll() or something
       // take a bunch of photos and videos
+      .then(function onSuccess () {
+        for (let i = 0; i < 3; i++) {
+        testClient.takepicture()
+        expectedImageCount++
+        totalEntryCount+=2
+        }
+      }).then(validate.done(res.body, schema.names.commandTakePicture))
+      .then(testClient.bublCaptureVideo())
```

Comment:
what do you think of my approach of capturing a bunch of images and videos in the before rather then doing it in every test? any foreseeable problems?